src/shared/bg_public.h

```diff
@@ -1127,10 +1127,24 @@ enum meansOfDeath_t
 // TODO: move other button definitions into gamelogic
 enum buttonNumber_t
 {
-  BUTTON_ATTACK3 = 9,
-  BUTTON_DECONSTRUCT = 13,
+	BTN_ATTACK       = 0,
+	// defined by daemon
+	BTN_TALK         = BUTTON_TALK, // = 1, // disables actions
+	BTN_USE_HOLDABLE = 2,
+	BTN_GESTURE      = 3,
+	// defined by daemon
+	BTN_WALKING      = BUTTON_WALKING, // = 4, // walking can't just be inferred from MOVE_RUN
```

Comment:
Actually BUTTON_WALKING is used directly by pmove to control speed. And there isn't anything anywhere called MOVE_RUN. So better to remove this comment.